rinex/src/meteo/mod.rs

```diff
@@ -14,3 +14,156 @@ pub struct HeaderFields {
     /// Sensors that produced the following observables
     pub sensors: Vec<sensor::Sensor>,
 }
+
+use crate::prelude::Epoch;
+
+/// Meteo RINEX record iteration methods.
+/// Faillible: if used on other RINEX types
+pub trait MeteoIter {
```

Comment:
You might want to pick a different name like `MeteoSource`. I'd expect `MeteoIter` to be a manually written iterator (struct) over meteo data.